static/lang/re-en.json

```diff
@@ -3359,6 +3359,9 @@
                 "HumbugPocket": {
                     "ConcealAnObjectLabel": "Humbug Pocket (Conceal an Object in the Pocket)"
                 },
+                "KaStone": {
+                    "Label": "vs. Death and Void effects"
+                },
```

Comment:
We have a generic section for this kind of thing

Reply:
Label and localization have been updated